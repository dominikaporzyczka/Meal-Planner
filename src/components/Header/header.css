nav {
  width: 100%;
  height: 40px;
  display: flex;
  align-items: center;
  background-color: #1abc9c;
 }
 
 .nav-link {
  text-decoration: none;
  padding: 10px 20px;
  color: #fff;
 }