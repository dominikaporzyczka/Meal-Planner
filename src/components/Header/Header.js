import React, { Component } from 'react';
import { NavLink } from 'react-router-dom';

import './header.css';

export class Header extends Component {
  render() {
    return (
      <nav>
        <NavLink to='/' className="nav-link">Logo</NavLink>
        <NavLink to='/recipes' className="nav-link">Recipes</NavLink>
        <NavLink to='/planner' className="nav-link">Your Meal Planner</NavLink>
      </nav>
    );
  }
};
