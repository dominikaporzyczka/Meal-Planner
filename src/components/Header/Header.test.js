import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter as Router } from 'react-router-dom';

import { Header } from './Header.js';

describe('Header', () => {
  it('should match snapshot', () => {
    const header = renderer
      .create(
        <Router>
          <Header />
        </Router>
      ).toJSON();

    expect(header).toMatchSnapshot();
  });
});
