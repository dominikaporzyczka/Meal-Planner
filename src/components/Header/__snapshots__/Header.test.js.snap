// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should match snapshot 1`] = `
<div>
  <nav>
    <a
      aria-current="page"
      className="active"
      href="/"
      onClick={[Function]}
      style={Object {}}
    >
      Logo
    </a>
    <a
      aria-current="page"
      className="active"
      href="/"
      onClick={[Function]}
      style={Object {}}
    >
      Recipes
    </a>
    <a
      aria-current={null}
      href="/planner"
      onClick={[Function]}
    >
      Planner
    </a>
  </nav>
</div>
`;
